src/nbody/barnes_hut.py

```diff
@@ -0,0 +1,373 @@
+from __future__ import annotations
+
+import csv
+from dataclasses import dataclass
+
+import numpy as np
+import Path
```

Comment:
```suggestion
from pathlib import Path
```

Is this an import from pathlib? Maybe this should be changed to: